fetch.bs

```diff
@@ -1038,6 +1041,39 @@ run these steps:
  <li><p>Return <var>values</var>.
 </ol>
 
+<p>To determine if a <a>byte sequence</a> <var>value</var> is a
+<dfn>simple range header value</dfn>, perform the following steps. They return a <a>boolean</a>.
+
+<ol>
+ <li><p>Let <var>data</var> be the <a>isomorphic decoding</a> of <var>value</var>.
+
+ <li><p>If <var>data</var> does not <a for=string>start with</a> "<code>bytes </code>", then return false.
```

Comment:
While I agree with you on wrapping, until we change it I do think we should maintain consistency.

Reply:
Yeah, just forgot 😄